### ISSUE TITLE
bounds property does nothing on Android

### ISSUE THREAD

**Issue by JasperVercammen** [rnmapbox/maps#2071]
**Describe the bug**  
Using the `bounds` property on the `Camera` does not seem to work on Android.

**To Reproduce**  
Clone and install the example app. Open the camera menu. Choose the Fit (Bounds, Center/Zoom, Padding) example.
Try the 2 "(bounds)" buttons, these will not trigger any actions.

Or use following snippet example:
```js
import React, { useState } from 'react';
import { Button, View } from 'react-native';
import Mapbox from '@rnmapbox/maps';

import { styles } from './map.styles';

Mapbox.setAccessToken('XXX');

const antwerpBounds = { ne: [4.500734, 51.347317], sw: [4.240822, 51.1462263] };
const belgiumBounds = { ne: [6.15665815596, 51.4750237087], sw: [2.51357303225, 49.5294835476] };

const Map = () => {
  const [bounds, setBounds] = useState(antwerpBounds);

  return (
    <View style={{ flex: 1 }}>
      <View style={{ flexDirection: 'row', marginTop: 40 }}>
        <Button onPress={() => setBounds(antwerpBounds)} title="Antwerp" />
        <Button onPress={() => setBounds(belgiumBounds)} title="Belgium" />
      </View>
      <Mapbox.MapView style={{ flex: 1 }} styleURL={Mapbox.StyleURL.Satellite}>
        <Mapbox.Camera animationDuration={500} bounds={bounds} maxZoomLevel={18} minZoomLevel={2} />
      </Mapbox.MapView>
    </View>
  );
};

export default Map;
```

**Expected behavior**  
The map should center and zoom the map to fit inside the bounding box.

**Actual behavior**  
Setting/changing the property does nothing

**Screenshots**  
![bounds_android](https://user-images.githubusercontent.com/2750271/180997562-3f14fb9f-3a4a-4e7a-80d4-deef94233a64.gif)

**Versions**  
 - Platform: Android
 - Platform OS: Tested on Android 12
 - Device: Pixel6Pro
 - Emulator/ Simulator: yes
 - Dev OS: OSX 12.3
 - @rnmapbox/maps Version 10.0.0-beta.18
 - React Native Version 0.66 and 0.68

**Comment by mfazekas** [comment]
Wasn't it fixed with https://github.com/rnmapbox/maps/pull/2015?!

**Comment by mfazekas** [comment]
I assume this was fixed with #2015. Please comment and clean up example (remove `import { styles } from './map.styles';` for example) if this issue still exists

**Comment by JasperVercammen** [comment]
Seems to work with version `beta.19`, although it drops a lot of errors/warnings in the example project on Android. I can't get it to work in my real project, so I assume there are more things changed.. Need to figure out if it's because of my project, or it's something in the lib, but I will open a new issue if that's the case.
